src/cljs/webchange/admin/schools/views.cljs

```diff
@@ -0,0 +1,65 @@
+(ns webchange.admin.schools.views
+  (:require [re-frame.core :as re-frame]
+            [webchange.editor-v2.components.card.views :refer [list-card] :as card]
+            [cljs-react-material-ui.reagent :as ui]
```

Comment:
do not use material-ui. we are going to get rid of it
use components from webchange/ui_framework/components/index.cljs instead